### PR TITLE
[ST-5182] Removing reference to docs-independent

### PR DESCRIPTION
**Pull request by kelvinL3**
### Description 

_What behavior does this PR change, and why?_

### Author Validation

_Describe the validation already done, or needs to be done, by the PR submitter._

### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._

### CODE REVIEW

**Review by rspurgeon — APPROVED** [review]
LGTM

File: docs/settings.sh [hunk 1]
```diff
@@ -52,7 +52,7 @@ DOCS_COMPONENT_PAGE_NAME="ansible"
 # Production S3 bucket that contains our production documentation repository.
 #
 # NOTE: You should never need to modify this setting!
-DOCS_BUCKET_PRODUCTION="docs-independent.confluent.io"
```
Comment (joel-hamill) [inline docs/settings.sh:55]: this is throwing an error when I try to run the local build:

```
source ./settings.sh && source ~/.virtualenvs/cp-demo-docs/bin/activate
ERROR: DOCS_BUCKET_PRODUCTION must be set

```

**Review by joel-hamill — COMMENTED** [review]
@kelvinL3 please test the local build before merging this change. as-is, this will break the local build.

File: docs/settings.sh (hunk 1 above)
Comment (joel-hamill) [reply docs/settings.sh:55]: @kelvinL3 nevermind, user error. invoking using `make livehtml` works fine (as described in the [README](https://github.com/confluentinc/cp-demo/blob/6.1.1-post/docs/README.md)). thanks.

**Pull request merged**